### ISSUE TITLE
Trailing space in CAP NAK

### ISSUE THREAD

**Issue by progval** [kaniini/mammon#54]
At the end of `CAP NAK` responses, Mammon adds a trailing space:
- https://github.com/mammon-ircd/mammon/blob/ab9a4e35ff292be7bb2b8cffe351042ebc6cd1a0/mammon/capability.py#L101
- https://github.com/mammon-ircd/mammon/blob/ab9a4e35ff292be7bb2b8cffe351042ebc6cd1a0/mammon/capability.py#L126

Why?

Also, [the spec](http://ircv3.net/specs/core/capability-negotiation-3.1.html#the-cap-nak-subcommand) states “The argument of the NAK subcommand MUST consist of at least the first 100 characters of the capability list in the REQ subcommand which triggered the NAK.”, and this space is not in the original message, so to my understanding, this is invalid.

**Comment by progval** [comment]
Note: git blame says: https://github.com/mammon-ircd/mammon/commit/45b0c5ff34b494ce283f5599eb05d545ab69d6ff

**Comment by DanielOaks** [comment]
If mIRC's fixed their client, we can take that out. We'll do some testing and see if it's still required!

**Comment by Mikaela** [comment]
I think this is breaking ZNC as when I connect I see:

```
2015-12-20 09:46:16+0200 -- *: C :Unknown command
2015-12-20 09:46:16+0200 -- *: AP :Unknown command
2015-12-20 09:46:16+0200 -- Welcome to the dereferenced.org IRC Network, Mikaela!mikaela@Ace-of-Diamonds.GirlsLikeUs.mikaela.info
```

And when I run `/cap` with WeeChat, I see just list of what ZNC supports.

```
2015-12-20 09:50:34+0200 -- irc: client capability, server supports: batch cap-notify echo-message multi-prefix server-time userhost-in-names znc.in/batch znc.in/playback znc.in/self-message znc.in/server-time-iso
2015-12-20 09:50:34+0200 -- irc: client capability, currently enabled: cap-notify multi-prefix server-time userhost-in-names
```

**Comment by Mikaela** [comment]
No, sorry, I missed the "NAK" in the tittle, I am opening a separate issue.
